spec/features/admin/dashboard_spec.rb

```diff
@@ -49,10 +48,9 @@
       login_page.do_login(owner.email, owner.password)
 
       expect(page).to have_content("Allow access")
-      expect(page).to have_content("Deny access")
+      expect(page).to have_selector("i.fa-times")
```

Comment:
Can use `have_selector 'h1#hola', text: 'Welcome'` to check the selector's content.